.github/workflows/delete-old-branches.yaml

```diff
@@ -0,0 +1,39 @@
+name: Delete old branches
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+permissions:
+  contents: read
```

Comment:
move the contents permission to inside the job

and in the top level do the following

```
permissions: {}
```